leapp/logger/__init__.py

```diff
@@ -52,39 +52,66 @@ def _remote_emit(self, log_data):
 
 
 def configure_logger(log_file=None):
-    global _logger
-    if not _logger:
+    """
+    Configure loggers as per the description below.
+
+    logger: root
+      level: DEBUG
+      handler: StreamHandler
+        level: based on the debug/verbosity options
+      handler: LeappAuditHandler
+        level: DEBUG
+      logger: urllib3
+        level: WARN
+      logger: leapp
+        level: NOTSET
+        handler: FileHandler
+          level: DEBUG
+
+    :return: The 'leapp' logger
+    """
+    global _leapp_logger
+    if not _leapp_logger:
+        _leapp_logger = logging.getLogger('leapp')
+
         log_format = '%(asctime)s.%(msecs)-3d %(levelname)-8s PID: %(process)d %(name)s: %(message)s'
```

Comment:
I think fo ruser would be convininent to see that this time is UTC time
```suggestion
        log_format = '%(asctime)s.%(msecs)-3d UTC %(levelname)-8s PID: %(process)d %(name)s: %(message)s'
```